### ISSUE TITLE
Linux: Notifications Accessible View only opens when focusing with mouse

### ISSUE THREAD

**Issue by chrmarti** [microsoft/vscode#191705]
Testing #191344

After focusing the notification by keyboard I cannot open the Accessible View. Only after clicking into it with the mouse cursor does the Accessible View open.

(Today's Insiders build on Linux.)

**Comment by meganrogge** [comment]
@bpasero that means that this context key is false on linux.

https://github.com/microsoft/vscode/blob/2af3045474f52bad8f14f01b09acfd5912e7fb5a/src/vs/workbench/contrib/accessibility/browser/accessibilityContributions.ts#L236

have you seen reports of this/ do you know why that would be the case?

**Comment by bpasero** [comment]
I cannot reproduce.

**Comment by chrmarti** [comment]
Just found that it only repros when TAB-ing to the notification when it is first shown (before it disappears), not when opening the notification center.

**Comment by bpasero** [comment]
@meganrogge its not the context actually, that one is set fine. But its this method:

https://github.com/microsoft/vscode/blob/ceac8ab3a335710d1ededb4af78a690af1364f2c/src/vs/workbench/browser/parts/notifications/notificationsCommands.ts#L71-L74

Even though the list has keyboard focus, there is no focussed element because the entire list is focused, not just one item. This is actually a consequence of using a list and I have https://github.com/microsoft/vscode/issues/182973 opened to actually drop usage of `List` for notifications. 

If you feel strong about it, feel free to adjust `getNotificationFromContext`. Maybe if the list has focus we should just always return the first element. I can review such a change.

**Comment by bpasero** [comment]
Actually this change was easy enough to do: https://github.com/microsoft/vscode/pull/191888
